backend/src/main/java/com/woowacourse/thankoo/common/schedule/MeetingScheduleTask.java

```diff
@@ -0,0 +1,24 @@
+package com.woowacourse.thankoo.common.schedule;
+
+import com.woowacourse.thankoo.meeting.domain.MeetingScheduleRepository;
+import com.woowacourse.thankoo.meeting.domain.MeetingStatus;
+import java.time.LocalDate;
+import lombok.RequiredArgsConstructor;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+@Component
+@RequiredArgsConstructor
+public class MeetingScheduleTask {
+
+    public static final LocalDate COMPLETE_JOB_DATE = LocalDate.now().minusDays(1L);
```

Comment:
```suggestion
    public static final LocalDate COMPLETE_JOB_DATE = LocalDate.now().minusYears(1L);
```

위와 같이 되어야 하는 것 아닐까용?